rtl/eclrtl/eclinclude4.hpp

```diff
@@ -46,11 +46,7 @@
 #include <math.h>
 
 #define CHEAP_UCHAR_DEF
-#if _WIN32
-typedef wchar_t UChar;
-#else //_WIN32
-typedef unsigned short UChar;
-#endif //_WIN32
+typedef cheap_uchar_t UChar;
```

Comment:
How will this be defined when compiling a query with eclcc, if the default doesn't match the value within ICU?